### PR TITLE
Action workflow: Prevent "E2E port 8084 kill step" to break flow on exitcode 1

### PR DESCRIPTION
**Pull request by kallehauge**
## Description

When trying to run our GitHub Action workflows locally via [`act`](https://github.com/nektos/act), I get the following error which prevents the next step(s) from being executed.

```
[E2E Tests/Build Container and Run E2E Tests]   ❓  ::group::Kill webserver running on port 8084
[E2E Tests/Build Container and Run E2E Tests]   ❌  Failure - Main Kill process running on port 8084 to unblock it
[E2E Tests/Build Container and Run E2E Tests] exitcode '1': failure
[E2E Tests/Build Container and Run E2E Tests] 🏁  Job failed
Error: Job 'Build Container and Run E2E Tests' failed
```

### Suggested changes

The issue is that `fuser` tries to kill anything that already runs on port `8084` (which we want to use for the website to run our E2E tests on).
If nothing is running on that specific port, then we get `exitcode: 1` back which just means "nothing to shut down".

TL;DR: Code 1 means that we're ready to proceed to next step since the port is available, so we need to update our bash code to accept "code 1" (we do this by catching `code 1` and returning `code 0`), but still "throw" all other error codes to prevent false positives from our change.

#### `fuser` exit code meanings:

* **0**: At least one process has been found and successfully terminated.
* **1**: No processes were found using the specified files or sockets.
* **2**: An error occurred.

### Related issue(s)

- Related to #2640 

### Steps to reproduce & screenshots/GIFs

* Install `act` locally ([you can use brew](https://formulae.brew.sh/formula/act))
* In CLI, be in the root of this repository and run `act`
    * You can use `act --container-architecture linux/amd64` if you're on a silicone Mac
 * Verify that you continue to the next step (named "`Build WCS&T`").
     * It's not important what happens in this step. We just need to reach it.

### Checklist

- [ ] unit tests

- [ ] `changelog.txt` entry added
- [ ] `readme.txt` entry added

### CODE REVIEW

**Comment by kallehauge** [comment]
I'm merging the PR since our automated tests are still completing, and I'm getting ` ✅  Success - Main Kill process running on port 8084 to unblock it` locally.
